src/source-pfring.c

```diff
@@ -532,9 +531,8 @@ TmEcode ReceivePfringThreadInit(ThreadVars *tv, const void *initdata, void **dat
 
     opflag = PF_RING_PROMISC;
 
-    /* if suri uses VLAN and if we have a recent kernel, we need
-     * to use parsed_pkt to get VLAN info */
-    if ((! ptv->vlan_disabled) && SCKernelVersionIsAtLeast(3, 0)) {
+    /* if we have a recent kernel, we need to use parsed_pkt to get VLAN info */
```

Comment:
this should be in its own commit

Reply:
Do you mean a fourth commit in addition to the list above?

Reply:
This is what I meant when referring pfring. The removal of ```SCKernelVersionIsAtLeast(3, 0)) {``` seems unrelated to the rest of the changes, thus must go in it's own commit